fontforge/python.c

```diff
@@ -838,15 +838,10 @@ static PyObject *PyFF_UnicodeBlockNameFromLib(PyObject *UNUSED(self), PyObject *
     return( ret );
 }
 
+/* Deprecated */
 static PyObject *PyFF_UnicodeNamesListVersion(PyObject *UNUSED(self), PyObject *UNUSED(args)) {
-/* If the library is available, then return the Nameslist Version number */
-    char *temp;
-
-    if ( (temp=unicode_library_version())==NULL ) {
-	temp=malloc(1*sizeof(char)); *temp='\0';
-    }
-    PyObject *ret=Py_BuildValue("s",temp); free(temp);
-    return( ret );
+    PyErr_Warn(PyExc_DeprecationWarning, "UnicodeNamesListVersion is deprecated and will be removed in a future release");
+    return Py_BuildValue("s","20190701");
```

Comment:
This function returns the version of Unicode that the names list was copied from (currently, `"Nameslist-Version: 12.1"`). With this patch, it returns the version of what used to be libuninameslist. Why change it?

Reply:
It sounds like there may have been confusion about the role of this function. The internal FontForge function for this string is `unicode_library_version()` so that confusion is understandable.  

It's frankly bizarre that this function is returning "Nameslist-Version: 12.1" instead of "12.1", but I guess there's little reason to change it now. 

@khaledhosny can we just leave the function chain as it is, or just call `uniNamesList_NamesListVersion()` directly?

Reply:
The version was removed because it was hard-coded in the `configure.ac` file. If we really want it, then `buildnameslist.c` needs to be modified to read it from the `NamesList.txt` file.

Reply:
Yeah - as with libspiro, all info was put at the top of configure.ac. This could be repeated in Fontforge's configure.ac, unfortunately 20190701 is incorrect. The info given was supposed to be the version of NamesList.txt, which is 12.1.